arangod/Cluster/ClusterMethods.cpp

```diff
@@ -1350,6 +1350,7 @@ futures::Future<OperationResult> countOnCoordinator(
 
 futures::Future<metrics::RawDBServers> metricsOnCoordinator(
     NetworkFeature& network, ClusterFeature& cluster) {
+  LOG_TOPIC("badf0", INFO, Logger::CLUSTER) << "Start collect metrics";
```

Comment:
```suggestion
  LOG_TOPIC("badf0", TRACE, Logger::CLUSTER) << "Start collect metrics";
```
Suggest to lower log level  as this INFO gives us not much info in general.